jina/proto/jina.proto

```diff
@@ -183,6 +183,25 @@ message Envelope {
 
     repeated uint32 num_part = 8; // the number of partial messages
 
+    string request_type = 9; // type of the request: TrainRequest, IndexRequest, SearchRequest, ControlRequest
+
+    bool check_version = 10; // check local Protobuf version on every Pod that this message flows to
```

Comment:
What about putting this into the `Flow`. Adding this to the `Envelop` does not seem to be necessary.
```suggestion
    bool check_version = 10; // check local Protobuf version on every Pod that this message flows to
```

Reply:
see `ProtoMessage` for the reason of this design